variants/vmware-dev/Cargo.toml

```diff
@@ -8,6 +8,7 @@ build = "build.rs"
 exclude = ["README.md"]
 
 [package.metadata.build-variant]
+image-format = "vmdk"
```

Comment:
I think this means that local testing in qemu would need a conversion to qcow2 first, since you mentioned that VMDKs only run in read-only mode.  (I'm thinking it's easier for some people to run in qemu than vmware locally.)

I'm wondering if we should make this a list instead of a single entry, so we could say `["raw", "vmdk"]` here and get the benefits of both?

Reply:
The alternative format is `img.lz4`, which also needs to be "converted" before it can be used under QEMU. I'd rather keep this as the primary format and let people convert to others as needed.

Reply:
> I'd rather keep this as the primary format and let people convert to others as needed.

Including those that need vmdk?

Reply:
If we went with a list, we could add a third option `raw_uncompressed` to represent `img` for testing.

Reply:
> Including those that need vmdk?  (Or does "this" mean vmdk?)

I did mean "vmdk" - we should output the (compressed) format that pubsys expects, and let other use cases manipulate it as needed.